config/application.rb

```diff
@@ -38,6 +38,8 @@ class Application < Rails::Application
     # Eager-load list_import files
     import_dirs = Dir['lib/list_import'].map { |d| File.expand_path(d) }
     config.eager_load_paths += import_dirs
+    # Include all services files
+    config.eager_load_paths += Dir["#{config.root}/app/services/**/"]
```

Comment:
I don't think you need this, Rails 4+ automatically adds any directories directly under `app` to this, so `app/services` is eager loaded by default (and we add `app/*/concerns` ourselves)

Doesn't look like you have anything that would rely on this, if I'm not mistaken